{{cookiecutter.project_slug}}/compose/local/docs/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apt-get update \
+    # dependencies for building Python packages
+    && apt-get install -y build-essential \
+    # psycopg2 dependencies
+    && apt-get install -y libpq-dev \
+    # Translations dependencies
+    && apt-get install -y gettext \
+    # Enable Sphinx output to latex and pdf
+    && apt-get install -y texlive-latex-recommended \
+    && apt-get install -y texlive-fonts-recommended \
+    && apt-get install -y texlive-latex-extra \
+    && apt-get install -y latexmk \
+    # cleaning up unused files
+    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+    && rm -rf /var/lib/apt/lists/*
+
+# Requirements are installed here to ensure they will be cached.
+COPY ./requirements /requirements
+# All imports needed for autodoc.
+RUN pip install -r /requirements/local.txt
+RUN pip install -r /requirements/production.txt
```

Comment:
Sphinx was getting build errors without some production dependencies.

Reply:
That's not ideal, but not too bad either.

Reply:
If this is not too much trouble can we dig into this?

Reply:
It is because sphinx needs ALL imports to work -- the environment it is run in must have all dependencies installed. The only way to get around this is to exclude files from sphinx's import and that really leads down a messy, impractical path that works against how it was designed.